.stylelintrc.json

```diff
@@ -9,6 +9,12 @@
         "ignoreProperties": ["composes"]
       }
     ],
+    "value-keyword-case": [
+      "lower",
+      {
+        "ignoreProperties": ["composes"]
```

Comment:
More info at: https://stylelint.io/user-guide/rules/value-keyword-case
Since we use CSS modules, we need to ignore `composes`, otherwise we get a bunch of these errors:

![image](https://user-images.githubusercontent.com/127199/74947473-c3591980-53af-11ea-9175-4e4bb50bd1b9.png)